python/tests/integration/template_engine/tests.txt

```diff
@@ -0,0 +1,73 @@
+# About this file:
+# - The file is used by PromptTemplateEngineTests.cs to test SK template language.
+# - By using a TXT file there is no ambiguity caused by C#/Python escaping syntax.
+# - Empty lines and lines starting with "#" are ignored.
+# - Lines are NOT trimmed, there might be empty spaces at the end on purpose.
+# - The file contains multiple test cases.
+# - Each test case consists of two lines:
+#   - line 1: the template to render
+#   - line 2: the expected result after rendering
+# - If a template is invalid, line 2 contains the value "ERROR", e.g. a TemplateException is expected.
```

Comment:
just checking, is the exception name still the same?

Reply:
Updated!